tests/Integration/Behaviour/Features/Scenario/Product/update_pack.feature

```diff
@@ -73,9 +73,9 @@ Feature: Add product to pack from Back Office (BO)
       | product  | quantity |
       | product2 | 5        |
     Then product "productPack1" type should be pack
-    And pack "productPack1" should contain products with following quantities:
-      | product  | quantity |
-      | product2 | 5        |
+    And pack "productPack1" should contain products with following details:
+      | product  | combination | name             | quantity |
+      | product2 |             | shady sunglasses | 5        |
```

Comment:
Here you need to add test for image url, you can inspire yourself from the related product tests:
https://github.com/PrestaShop/PrestaShop/blob/27291668a782ac9917a9cea2e6c0c88d002d6a76/tests/Integration/Behaviour/Features/Context/Domain/Product/RelatedProductsFeatureContext.php#L126-L135